src/test/ui/issue46332.rs

```diff
@@ -0,0 +1,14 @@
+// Original problem is Levenstein distance.
+
+fn TyUint() {
```

Comment:
(An enum variant rather than a function is closer to the spirit of the example in the issue (which isn't to say that we shouldn't test functions, too, but those are `snake_case_in_standard_style`, unlike `TyUint`).)